nucliadb/nucliadb/writer/api/v1/services.py

```diff
@@ -90,6 +100,100 @@ async def set_entities(request: Request, kbid: str, group: str, item: EntitiesGr
         )
 
 
+@api.post(
+    f"/{KB_PREFIX}/{{kbid}}/entitiesgroups",
```

Comment:
If you are adding new services, they need to add them to the virtual services yamls to be reachable in the hosted version of nucliadb

Reply:
If I'm right, they are already covered by this: https://github.com/nuclia/nucliadb/blob/bb88f30df81b095ffe227c4356ac723d3a07fca2/charts/nucliadb_writer/templates/writer.vs.yaml#L18